cli/src/pcluster/cli/entrypoint.py

```diff
@@ -0,0 +1,145 @@
+            "launching and management of HPC clusters in the AWS cloud.",
+            epilog='For command specific flags, please run: "pcluster [command] --help"',
+        )
+        subparsers = self._parser.add_subparsers(dest="command")
+        subparsers.required = True
+        for command_class in CliCommand.__subclasses__():
+            if not isabstract(command_class):
+                command_class(subparsers)  # pylint: disable=abstract-class-instantiated
+
+        dcv = subparsers.add_parser(
+            "dcv",
+            help="The dcv command permits to use NICE DCV related features.",
+            epilog='For dcv subcommand specific flags, please run: "pcluster dcv [subcommand] --help"',
+        )
+        dcv_subparsers = dcv.add_subparsers(dest="command")
+        dcv_subparsers.required = True
+        for dcv_command_class in DcvCommand.__subclasses__():
+            dcv_command_class(dcv_subparsers)
+
+    def handle_command(self):
+        """Handle CLI command execution."""
+        try:
+            args, extra_args = self._parser.parse_known_args()
+
+            if args.debug:
+                logging.getLogger("pcluster").setLevel(logging.DEBUG)
+            LOGGER.debug("Parsed CLI arguments: args(%s), extra_args(%s)", args, extra_args)
+
+            # TODO: remove this logic from here
+            # set region in the environment to make it available to all the boto3 calls
+            if "region" in args and args.region:
+                os.environ["AWS_DEFAULT_REGION"] = args.region
+
+            if not args.expects_extra_args and extra_args:
+                self._parser.print_usage()
+                print("Invalid arguments %s" % extra_args)
+                sys.exit(1)
+
+            args.func(args, extra_args)
+        except NoCredentialsError:  # TODO: remove from here
+            LOGGER.error("AWS Credentials not found.")
+            sys.exit(1)
+        except KeyboardInterrupt:
+            LOGGER.debug("Received KeyboardInterrupt. Exiting.")
+            sys.exit(1)
+        except Exception as e:
+            LOGGER.exception("Unexpected error of type %s: %s", type(e).__name__, e)
+            sys.exit(1)
+
+    @staticmethod
+    def _config_logger():
+        logfile = get_cli_log_file()
+        logging_config = {
+            "version": 1,
+            "disable_existing_loggers": True,
+            "formatters": {
+                "standard": {"format": "%(asctime)s - %(levelname)s - %(module)s - %(message)s"},
+                "console": {"format": "%(message)s"},
+            },
+            "handlers": {
+                "default": {
+                    "level": "DEBUG",
+                    "formatter": "standard",
+                    "class": "logging.handlers.RotatingFileHandler",
+                    "filename": logfile,
+                    "maxBytes": 5 * 1024 * 1024,
+                    "backupCount": 1,
```

Comment:
not related to your patch but what about increasing the number of backups?

Reply:
Increased to 3